base/doc/ltnews41.tex

```diff
@@ -743,6 +743,20 @@ \subsection{Fully expand the arguments of \cs{counterwithin} and \cs{counterwith
 \githubissue{1675}
 
 
+\subsection{Ensure that late \cs{write} commands aren't lost}
+
+A non-\cs{immediate} \cs{write} command that is used after the final
+page has been shipped out is never written because it waits for
+another \cs{shipout} to happen.  After the last page has been shipped
+out, we therefore force further \cs{write} calls to be always
+\cs{immediate} to ensure that there are written even though we are not
+going to going to ship out any more pages. This change of behavior is
```

Comment:
. . . ; this ensures that they get written even though we are not going to ship out . . .